pkg/cluster/api/pdapi.go

```diff
@@ -732,6 +733,42 @@ func (pc *PDClient) GetTiKVLabels() (map[string]map[string]string, error) {
 	return locationLabels, nil
 }
 
+// GetTiKVStoreStateAndLabels implements TiKVLabelProvider
+func (pc *PDClient) GetTiKVStoreStateAndLabels() ([]map[string]string, error) {
```

Comment:
There is already a `GetLocationLabels()` function to query TiKV labels, and store statuses can be fetched with instance's `Status()` method. What's the purpose of adding this new function? Is it necessary?

Reply:
origin Display function isn‘t very easy to change，so add new function and flag

Reply:
So why adding a whole new display mode from the fist place? Why not adding them to the current output?

Reply:
`GetLocationLabels()` function isn't macth my demand，I want get all machine【machine ip，not with port】 labels and region、leader etc info...

Reply:
But... again... why? Why would adding a whole new table with such information be necessary, and why adding them to the current display table would not work? What's the usage senario of this new panel?

Reply:
em... GetLocationLabels() state only not up and I do not confirm whether there are special usages elsewhere. If GetLocationLabels() is not special, can consider combining the two into one

Reply:
`GetLocationLabels()` is used in `display.go` and `scale_out.go` in `pkg/cluster/manager`, they all used to check for labels. So I think it's safe to adjust it to fit the display usage. It could also avoid redundant requests to PD.